includes/formatting.php

```diff
@@ -674,6 +674,51 @@ function give_get_cache_key( $action, $query_args ) {
 	return Give_Cache::get_key( $action, $query_args );
 }
 
+
+/**
+ * Get sanitized super global param
+ *
+ * @param string $type
+ *
+ * @return mixed
+ * @since 2.5.0
+ *
+ */
+function give_get_super_global( $type ) {
```

Comment:
What about a second param to get an array key from the super global like `$_POST['give-gateway']`?

Reply:
@DevinWalker I think, this is a good suggestion. I will update that.